aiogram/api/client/bot.py

```diff
@@ -101,12 +101,12 @@
 from .base import BaseBot
 
 
-class Bot(BaseBot):
+class Bot(BaseBot['Bot']):
     """
     Class where located all API methods
     """
 
-    @alru_cache()
+    @alru_cache()  # type: ignore
```

Comment:
alru_cache doesn't have typing :(

Reply:
No, it isn't solution, because strict mypy has setting about typed decorators and alru_cache has not typing.